README.md

```diff
@@ -88,6 +88,16 @@ workers:
   transfer: 10
 ```
 
+Excluding directories or files to be scraped from the source can be reached with specifying a regex in the job definition:
+```yaml
+jobs:
+  - from: http://de.archive.ubuntu.com/ubuntu/
+    to:   mirror/ubuntu-repos
+    excl: /sub_dir$|.gz$
+```
+
+This would exclude directories named `sub_dir` and files with extension `gz` on every level.
```

Comment:
Are you sure. I am using `directory.path`, not `directory.SourceURL()`. The trailing slash was not there in my testings.